sdk/core/core-http/src/policies/userAgentPolicy.ts

```diff
@@ -50,13 +53,22 @@ function getUserAgentString(
 
 export const getDefaultUserAgentHeaderName = getDefaultUserAgentKey;
 
+/**
+ * The default approach to generate user agents.
+ * Uses static information from this package, plus system information available based on the operative system running Node.js
```

Comment:
We can still get system information in the browser. This policy isn't runtime specific, but `getPlatformSpecificData` which it imports, is.

```suggestion
 * Uses static information from this package, plus system information available from the runtime.
```